src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,46 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int INDEX_OF_VALUE = 1;
+    private static final String CVS_SEPARATOR = ",";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        writeReport(toFileName, createAndGetReportFromFile(fromFileName));
+    }
+
+    private String createAndGetReportFromFile(String fileName) {
+        int supply = 0;
+        int buy = 0;
+
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fileName))) {
+            String readable;
+            while ((readable = bufferedReader.readLine()) != null) {
+                if (readable.contains("supply")) {
+                    supply += Integer.parseInt(readable.split(CVS_SEPARATOR)[INDEX_OF_VALUE]);
+                }
+                if (readable.contains("buy")) {
+                    buy += Integer.parseInt(readable.split(CVS_SEPARATOR)[INDEX_OF_VALUE]);
```

Comment:
let's save `readable.split(CVS_SEPARATOR)[INDEX_OF_VALUE]` into a variable and then reuse it